overlord/hookstate/hookmgr.go

```diff
@@ -70,7 +70,9 @@ type Handler interface {
 	Done() error
 
 	// Error is called if the hook encounters an error while running.
-	Error(err error) error
+	// The returned bool flag indicates if the original hook error should be
+	// ignored by hook manager.
+	Error(hookErr error) (ignoreHookError bool, err error)
```

Comment:
nitpick, so that the return argument name matches the input
```suggestion
	Error(hookErr error) (ignoreHookErr bool, err error)
```